src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
There is considerable amount of difference between `MetaFields` and `DataGUI` that I think its best to keep them separate.